ESP_Code/ESP-Code.ino

```diff
@@ -78,6 +82,8 @@ namespace {
         MINER_KEY
     );
 
+    static bool eth_connected = false;
```

Comment:
Shouldn't this be placed in an #ifdef USE_LAN block?

Reply:
Yes, that's true. Just give me a second.

I'm not really good at programming yet - please bear with me